docs/configuration/backends/docker.md

```diff
@@ -145,6 +145,18 @@ To enable constraints see [backend-specific constraints section](/configuration/
 
 ## Labels: overriding default behaviour
 
+!!! note
+    If you use a docker-compose file, labels should be defined in the `deploy` part of your service.
```

Comment:
Can precise two points : 
- This behavior is only enabled for docker-compose API 3+
- Add a ling to the docker-compose documentation : https://docs.docker.com/compose/compose-file/#labels-1